#goal definition
string motor_name
---
#result definition
string status
---
#feedback
int32 count